src/main/java/com/kodilla/ecommercee/controller/UserController.java

```diff
@@ -31,8 +33,8 @@ public UserDto blockUser(@RequestParam Long id) throws UserNotFoundException {
         return userMapper.userToUserDto(userService.blockUser(id));
     }
 
-    @GetMapping(value = "getToken")
-    public Long getToken(@RequestParam Long userId) throws UserNotFoundException, ExecutionException {
-        return GuavaCacheService.getUserCache().get(userService.getUserById(userId));
+    @PutMapping(value = "generateToken")
+    public void generateToken(@RequestParam Long userId) throws UserNotFoundException, ExecutionException {
+        userService.getUserById(userId).setToken(tokenService.getUserCache().get(userService.getUserById(userId)));
```

Comment:
Po co ta część?
```
userService.getUserById(userId).setToken(
```

Reply:
żeby ustawić token dla konkretnego uzytkownika po jego ID, zgodnie z specyfikacja projektu:
![image](https://user-images.githubusercontent.com/44532945/56222295-0c439180-606c-11e9-99d0-1f3b6a932fbe.png)
Jesli nie wten sposob to prosze o nakierowanie ;) @maciek-prokopiuk

Reply:
niepotrzebna... :)